src/sentry/static/sentry/app/views/ruleEditor/utils.js

```diff
@@ -0,0 +1,33 @@
+/*eslint no-use-before-define:0*/
+
+export function getHtmlForNode(node) {
```

Comment:
Why declare this up so high, requiring the `no-use-before-define` (or maybe some other things need that)?

Reply:
I find it a lot easier to read a file with the main function at the top and helper functions below. Updated to fit our linting rules though.